modules/sms_devel/tests/src/Functional/SmsDevelMessageTest.php

```diff
@@ -47,12 +46,12 @@ public function testSendSkipQueue() {
     $edit['skip_queue'] = TRUE;
 
     $this->drupalPostForm(Url::fromRoute('sms_devel.message'), $edit, t('Send'));
-    $this->assertResponse(200);
```

Comment:
I guess it's ok to remove deprecated code usages even though it introduces scope creep.

Reply:
I can commit the deprecation commit separately instead of squashing the whole thing.